test/packaging/ansible/roles/initial-install/tasks/config-Linux.yaml

```diff
@@ -0,0 +1,12 @@
+---
+
+- name: install infra-agent configuration file
+  template:
+    src: ../files/etc/newrelic-infra.yml.j2
```

Comment:
the file `newrelic-infra.yml.j2` is missing ☝️

Reply:
dammit! this was the cause https://github.com/newrelic/infrastructure-agent/blob/master/.gitignore#L13

Reply:
😅